BlazarData/src/main/java/com/hubspot/blazar/data/service/ModuleBuildService.java

```diff
@@ -47,8 +51,32 @@ public ModuleBuildService(ModuleBuildDao moduleBuildDao, ModuleDao moduleDao, Ev
     return moduleBuildDao.getByState(state);
   }
 
-  public List<ModuleBuild> getByModule(int moduleId) {
-    return moduleBuildDao.getByModule(moduleId);
+  public ModuleActivityPage getModuleBuildHistoryPage(int moduleId, Optional<Integer> maybeFromBuildNumber, Optional<Integer> maybePageSize) {
```

Comment:
Method name still has 'History' in it instead of activity. Maybe the dao methods should be changed too